pkg/logrusutil/logrusutil_test.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 func TestPropagateJSON(t *testing.T) {
-	loggerWithoutTs := func(output *bytes.Buffer) *logrus.Logger {
+	loggerWithoutTS := func(output *bytes.Buffer) *logrus.Logger {
```

Comment:
Not sure if we want to enable this rule

Reply:
> Not sure if we want to enable this rule

Why not? I assume this is the rule that requires all acronyms to be fully uppercase, so enforces `JSON`, and `IP` over `Json` and `Ip`.